frontend/components/MeetupCluster.vue

```diff
@@ -0,0 +1,235 @@
+        <div class="logo">
+          <img :src="pin.group_photo">
+          <span class="m"> Meetup </span>
+        </div>
+        <div class="details">
+          <v-layout
+            column
+            align-content-center
+          >
+            <v-flex class="title">
+              {{ pin.name }}
+            </v-flex>
+            <v-flex class="date">
+              {{ formatTooltipDate(pin) }}
+            </v-flex>
+          </v-layout>
+        </div>
+      </v-layout>
+    </map-marker>
+  </v-marker-cluster>
+</template>
+
+<script>
+import { mapGetters } from 'vuex';
+import { format } from 'date-fns';
+import MapMarker from './MapMarker.vue';
+
+export default {
+  components: {
+    MapMarker
+  },
+  props: {
+    showFloatingUi: {
+      type: Boolean,
+      default: false
+    }
+  },
+  data () {
+    return {
+      meetupIcon: null,
+      meetupEventIcon: null,
+      meetupIconSelected: null,
+      meetupEventIconSelected: null
+    };
+  },
+  computed: {
+    ...mapGetters({
+      meetupsGroups: 'events/getMeetups',
+      currentMeetup: 'events/getCurrentMeetup'
+    }),
+    clusterOptions () {
+      return {
+        disableClusteringAtZoom: 13,
+        spiderfyOnMaxZoom: false,
+        polygonOptions: {
+          stroke: false,
+          fillColor: '#42B883'
+        },
+        iconCreateFunction: cluster => {
+          const html = `<span>${cluster.getChildCount()}</span>`;
+          return L.divIcon({
+            className: `meetup-cluster-icon`,
+            html,
+            iconSize: [48, 58],
+            iconAnchor: [24, 58]
+          });
+        }
+      };
+    }
+  },
+  mounted () {
+    this.meetupIcon = new L.divIcon({ // eslint-disable-line
+      className: `custom-meetup-icon`,
+      iconSize: [40, 46],
+      iconAnchor: [20, 46]
+    });
+    this.meetupEventIcon = new L.divIcon({ // eslint-disable-line
+      className: `custom-meetup-icon meetup-event`,
+      iconSize: [34, 52],
+      iconAnchor: [17, 52]
+    });
+    this.meetupIconSelected = new L.divIcon({ // eslint-disable-line
+      className: `custom-meetup-icon meetup-selected`,
+      iconSize: [40, 46],
+      iconAnchor: [20, 46]
+    });
+    this.meetupEventIconSelected = new L.divIcon({ // eslint-disable-line
+      className: `custom-meetup-icon meetup-event meetup-selected`,
+      iconSize: [34, 52],
+      iconAnchor: [17, 52]
+    });
+  },
+  methods: {
+    openMeetupDetails (id) {
+      this.$router.push(`/meetup/${id}/`);
+    },
+    iconChooser (pin) {
+      if (pin.has_event_with_coords) {
+        return this.currentMeetup !== pin.id ? this.meetupEventIcon : this.meetupEventIconSelected;
+      }
+      return this.currentMeetup !== pin.id ? this.meetupIcon : this.meetupIconSelected;
+    },
+    formatTooltipDate (pin) {
+      if (pin && pin.event && pin.event.date) {
+        const date = format(pin.event.date, 'DD MMMM, YYYY');
+        return `${date} - ${pin.event.local_time}`;
```

Comment:
events should only show their date in their local timezone. event.date is now UTC, so you can use that, but you need to add event.utc_offset to it. The easiest tho is to use event.local_time & local_date.

Reply:
No is not, because of the different formatting of the date . Events at the moment show In the same date as meetup page like this

Reply:
@lordfuoco once again, event.date is the datetime in UTC, we need to show the Event's local time, which has an offset from UTC (which is also stored as data.utc_offset). But i would rather skip the calculation and show data.local_time + data.local_date.

Reply:
also, if you set your environment timezone to something else than UTC, then event.date will reflect that, so eg: "2018-07-24T20:00:00+02:00" is the date in my timezone 'Europe/Budapest' which is currently +02:00 from UTC.